lib/export-guest-list-excel.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+var fs = require('fs');
+var async = require('async');
+
+function exportGuestListExcel (args, callback) {
+  var seneca = this;
+  var eventId = args.eventId; 
+  var plugin = args.role;
+
+  async.waterfall([
+    retrieveUserData,
+    convertToExcel
+  ], function (err, csv) {
+    if (err) return callback(null, {error: err});
+    return callback(null, {
+      data: csv
+    });
+  });
+
+  function retrieveUserData (done) {
+    seneca.act({role: plugin, cmd: 'searchApplications', query: {eventId: eventId, status: 'approved', deleted: false}}, function (err, applications) {
+      if (err) return callback(err);
+      async.map(applications, function (application, cb) {
+        seneca.act({role: 'cd-profiles', cmd: 'list', query: {userId: application.userId}}, function (err, profiles) {
+          if (err) return cb(err);
+          var userProfile = profiles[0];
+          seneca.act({role: plugin, cmd: 'loadSession', id: application.sessionId}, function (err, session) {
+            if (err) return cb(err);
+            var user = {};
+            user['Session'] = session.name;
+            user['Name'] = userProfile.name;
+            user['Phone'] = userProfile.phone || '';
+            user['Email'] = userProfile.email || '';
+            user['Ticket Name'] = application.ticketName;
+            user['Ticket Type'] = application.ticketType;
+            return cb(null, user);
+          });
+        });
+      }, function (err, data) {
+        if (err) return callback(null, {error: err});
+        return done(null, data);
+      });
+    });
+  }
+
+  function convertToExcel (data, done) {
+    var writeStream = fs.createWriteStream("guestlist.xls");
```

Comment:
actually the CSV export function already exists.
I want to add this function specifically to create a table as excel because it then opens directly opens in excel.

with cvs you need to (1) download file, (2) open excel, (3) create empty excel, (4) import the csv in excel, (5) select how to convert csv data

I prefer 1 download instead of 1 download + 5 import steps....